File: Problems_and_Solutions/0167_Two-Sum-II_Input-array-is-sorted/Solution.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int lo = 0, hi = numbers.length - 1;
        while (lo < hi) {
            int sum = numbers[lo] + numbers[hi];
            if (sum == target) {
                return new int[] { ++lo, ++hi };
            } else if (sum > target) {
                hi--;
            } else {
                lo++;
            }
        }
        return null;

    }
}

File: Problems_and_Solutions/0167_Two-Sum-II_Input-array-is-sorted/Solution.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        lo, hi = 0, len(numbers) - 1
        while lo < hi:
            sum = numbers[lo] + numbers[hi]
            if sum == target:
                return [lo+1, hi+1]
            elif sum < target:
                lo += 1
            else:
                hi -= 1
        return None
        